server/autobuild

```diff
@@ -443,6 +443,32 @@ os_void() {
     return 0
 }
 
+# openSUSE (TODO: add support for micro versions)
+os_opensuse() {
+    if [ -f "/etc/os-release" ]; then
+        . /etc/os-release
+        if [ "$ID" != "opensuse-leap" ] && [ "$ID" != "opensuse-tumbleweed" ]; then
```

Comment:
If it is an OR though, then wouldn't it always return? If the ID is `opensuse-leap`, then it is not `opensuse-tumbleweed` so it would return. If the ID is `opensuse-tumbleweed`, then it is not `opensuse-leap` so it would also return.